### PR TITLE
added navigation. not sure if this is the most memory efficient way (…

### PR DESCRIPTION
**Pull request by juanyepesp**
…should ask)

btw sorry for not using commitizen, will do moving forward

### CODE REVIEW

File: FoodBookApp/UI/Views/ForYouView.swift [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+//
+
+import SwiftUI
+
+struct ForYouView: View {
+    var body: some View {
+        ZStack {
+            
+            Image(systemName: "star")
+                .foregroundColor(/*@START_MENU_TOKEN@*/.blue/*@END_MENU_TOKEN@*/)
```
Comment (mar-cas3) [inline FoodBookApp/UI/Views/ForYouView.swift:15]: im not clear on what @START_MENU_TOKEN does, please add an inline comment explaining (ideally everywhere its used)

**Review by mar-cas3 — CHANGES_REQUESTED** [review]

File: FoodBookApp/UI/Views/BookmarksView.swift [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+import SwiftUI
+
+struct BookmarksView: View {
+    var body: some View {
+        
+        ZStack {
+            
+            Image(systemName: "book")
+                .foregroundColor(/*@START_MENU_TOKEN@*/.blue/*@END_MENU_TOKEN@*/)
```
[the remark this answers is not in the document]
Comment (juanyepesp) [reply FoodBookApp/UI/Views/BookmarksView.swift:16]: yes! but i left it the same just for distinguishing between views right now. in the future they should display their actual content

[the remark this answers is not in the document]
Comment (mar-cas3) [reply FoodBookApp/UI/Views/BookmarksView.swift:16]: ahh okay, looks good then!

File: FoodBookApp/UI/Views/ForYouView.swift (hunk 1 above)
Comment (juanyepesp) [reply FoodBookApp/UI/Views/ForYouView.swift:15]: Besti i do NOT know that that is, i guess it looks like that in source code but in swift this is the syntax:

Screenshot 2024-02-22 at 7 28 20 AM

Comment (mar-cas3) [reply FoodBookApp/UI/Views/ForYouView.swift:15]: let me clone real quick and check to see what i get

Comment (mar-cas3) [reply FoodBookApp/UI/Views/ForYouView.swift]: yes on local its normal, should be some underlying syntax

**Review by mar-cas3 — APPROVED** [review]

**Comment by mar-cas3** [comment]
bottomnav

@juanyepesp adding just for tracebability

**Pull request merged**